src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static int index = 0;
+    private static String separator = "-------------------------------------------";
+    private static Task[] tasks = new Task[100];
+    public static void greet() {
+        String logo = "   ________                            \n"
+                + "  / ____/ /_  ___  ___  ____ ___  _____\n"
+                + " / /   / __ \\/ _ \\/ _ \\/ __ `__ \\/ ___/\n"
+                + "/ /___/ / / /  __/  __/ / / / / (__  ) \n"
+                + "\\____/_/ /_/\\___/\\___/_/ /_/ /_/____/  \n"
+                + "                                       \n"
+                + "-------------------------------------------\n";
+        String greet = "Hemlo from\n" + logo + "Whamt cam cheems do for you?\n"
+                + "-------------------------------------------";
+        System.out.println(greet);
+    }
+
+    public static void farewell() {
+        String farewell = "Goodbye. See you next time frem!\n" + "-------------------------------------------";
+        System.out.println(farewell);
+    }
+
+    public static void printList() {
+        System.out.println("Hemre are the tasks im your list:");
```

Comment:
Was initially confused if you made a typo; realized it was the bot's personality! Nice!